ocs_ci/deployment/helpers/icsp_parser.py

```diff
@@ -0,0 +1,119 @@
+import logging
+
+"""
+This module contains the ImageContentSourcePolicy parser
+Main task is to parse the ImageContentSourcePolicy cr in format, suitable for
+passing 'mirrors to sources' file to 'hcp' or 'hypershift' binaries while creating
+hosted cluster.
+"""
+
+logger = logging.getLogger(__name__)
+
+
+class ImageContentSourcePolicy:
+    def __init__(self, apiVersion=None, items=None, kind=None, metadata=None, **kwargs):
```

Comment:
Please add docstring to classes in this module.

Reply:
+ added